price-feeder/cmd/price-feeder.go

```diff
@@ -112,12 +114,18 @@ func priceFeederCmdHandler(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("failed to parse RPC timeout: %w", err)
 	}
 
+	// Gather pass via env variable || std input
+	keyringPass, err := getKeyringPassword()
```

Comment:
The SDK has a `input.GetString` method we can call. We can save on code if we just call that :)